client/src/components/content/Content.js

```diff
@@ -1,37 +1,15 @@
 import React, {Component} from 'react'
 import TopPanel from '../topPanel/TopPanel'
 import MessageList from '../message/MessageList'
-import {Loader} from '../ui'
-import {getReportsByTag} from '../../utils/serverApi'
 import './Content.scss'
 
 class Content extends Component {
-  state = {
-    loading: true,
-  }
-
-  async componentDidMount() {
-    const response = await getReportsByTag(this.props.tag)
-    this.setState({...response, loading: false})
-  }
-
```

Comment:
I think this loading state doesn't make much sense in this component now, that this component is only rendered once everything has been downloaded.